Microsoft.Azure.CosmosRepository.Samples/ServiceTier/ExampleService.cs

```diff
@@ -1,37 +1,36 @@
-﻿// Copyright (c) IEvangelist. All rights reserved.
-// Licensed under the MIT License.
-
-using System;
-using System.Collections.Generic;
-using System.Linq.Expressions;
-using System.Threading.Tasks;
-using Microsoft.Azure.CosmosRepository;
+﻿// Copyright (c) IEvangelist. All rights reserved. Licensed under the MIT License.
 
 namespace ServiceTier
 {
+    using System;
+    using System.Collections.Generic;
+    using System.Linq.Expressions;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.CosmosRepository;
+
     public class ExampleService : IExampleService
     {
-        readonly IRepository<Person> _personRepository;
+        private readonly IRepository<Person> personRepository;
```

Comment:
I prefer to omit default access modifiers, for example `private` on fields - this is again just a style preference.

Reply:
Those were put in by codemaid. I didn't hand enter those so I can flip the switch and revert those easily enough.